lib/utils.js

```diff
@@ -3,10 +3,10 @@ const jitter = (base, offset) => (
   base + Math.floor(Math.random() * Math.floor(offset))
 )
 
-// add a random number between 0 and 50 to the base number
-const jitter50 = (base) => jitter(base, 50)
+// add a random number between 0 and x to the base number
+const jitterx = (base, x) => jitter(base, x)
 
 module.exports ={
   jitter,
-  jitter50
+  jitterx
```

Comment:
how is `jitterx` different than `jitter`?

Reply:
jitterx uses jitter, it jitters the base with a random number from 0-x